docs-source/spring/content/infrastructure/database/_index.md

```diff
@@ -0,0 +1,103 @@
+
+By default, the Oracle Autonomous Database - Shared (ADB-S) is used for the  **Metadata Database**, however, there are other options including Bring Your Own (BYO).
+
+The following chart presents the options for the Metadata Database, based on the installation type:
+
+| Installation  | ADB-S | BYO ADB-S  | BYO BaseDB | BYO Containerized | BYO External |
+|---------------|-------|------------|------------|-------------------|--------------|
+| OCI Community | x     |            |            |                   |              |
+| OCI Standard  | x     | x          | x          |                   | x            |
+| Custom        | x     | x          | x          | x                 | x            |
+
+> For **custom installations**, including on-premises, it is the responsibility of the user to ensure network access controls to provide both operational access and security.  The Oracle Cloud Infrastructure (OCI) [Networking](../networking) setup can be used as a general template.
+
+# Bring Your Own Database - Standard Edition
+
+If you select the **Standard Edition** during installation, you can use a pre-created Oracle Database for the Oracle Backend for Spring Boot and Microservices **Metadata Database**.  
+
+The following are the minimum requirements for a BYO Oracle Database:
+
+* Version: 19c+
+* Bring Your Own Network with access to the Database Listener
+* Database User with appropriate privileges (see below)
+
+## Database User Privileges
+
+The database user for the the Oracle Backend for Spring Boot and Microservices **Metadata Database** is used to create other users and allow them to proxy through this user for database access.  While the `SYSTEM` or `ADMIN` (for ADB-S) will work, they are over-privileged and should not be used in production environments.
+
+It is recommended to create a user, in this example, named `OBAAS` with a default tablespace of `DATA`:
+
+```sql
+CREATE USER OBAAS IDENTIFIED BY "Welcome_12345";
+ALTER USER OBAAS QUOTA UNLIMITED ON DATA;
+GRANT ALTER USER TO OBAAS;
+GRANT CREATE USER TO OBAAS;
+GRANT CONNECT TO OBAAS WITH ADMIN OPTION;
+GRANT CREATE SESSION TO OBAAS WITH ADMIN OPTION;
+GRANT RESOURCE TO OBAAS WITH ADMIN OPTION;
+GRANT SELECT ON DBA_USERS TO OBAAS;
+GRANT CREATE ANY INDEX TO OBAAS;
+GRANT ALTER ANY TABLE TO OBAAS;
+GRANT COMMENT ANY TABLE TO OBAAS;
+GRANT CREATE ANY TABLE TO OBAAS;
+GRANT INSERT ANY TABLE TO OBAAS;
+GRANT SELECT ANY TABLE TO OBAAS;
+GRANT UPDATE ANY TABLE TO OBAAS;
+GRANT CREATE ANY SEQUENCE TO OBAAS;
+GRANT SELECT ANY SEQUENCE TO OBAAS;
+GRANT CREATE ANY TRIGGER TO OBAAS;
+-- Additional for PARSE
+GRANT SODA_APP TO OBAAS WITH ADMIN OPTION;
+GRANT CREATE TABLE TO OBAAS WITH ADMIN OPTION;
```

Comment:
does this user need aq_adm.... as well?

Reply:
@andytael .. do you have the AQ stuffs?

Reply:
@gotsysdba here it is:
```
grant aq_user_role to account;
grant execute on dbms_aq to account;
grant execute on dbms_aqadm to account;
grant execute on dbms_aqin to account;
grant execute on dbms_aqjms_internal to account;
```
Don't know why `dbms_aqjms_internal` is there?